kuma/misc/urls.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.conf.urls import url
+from django.views.static import serve
+
+from . import views
+
+
+url_patterns = [
+    # Serve the revision hashes.
+    url(r'^media/revision.txt$',
+        views.revision_hash,
+        name='misc.revision'),
+    url(r'^media/kumascript-revision.txt$',
+        views.kumascript_revision_hash,
+        name='misc.kumascript_revision'),
+
+    # Serve sitemap files for AWS (these are never hit in SCL3).
+    url(r'^sitemap.xml$',
+        serve,
+        {'document_root': settings.MEDIA_ROOT, 'path': 'sitemap.xml'},
+        name='misc.sitemap'),
+    url(r'^(?P<path>sitemaps/.+)$',
```

Comment:
``sitemap.xml`` and ``sitemaps/`` should be added to ``LANGUAGE_URL_IGNORED_PATHS `` in ``kuma/settings/common.py``, to avoid the locale redirect.